.gitignore

```diff
@@ -9,6 +9,7 @@ downloads
 eggs
 parts
 build
+venv
```

Comment:
Please avoid unrelated changes to the issue being addressed.

Reply:
venv is a common name for a virtualenv.

Reply:
I'm not saying otherwise.

I'm saying don't mix non-functional code changes in a functional PR. Please keep the change targeted to the issue at hand. If you want suggest other changes, please do so as separate commits so we aren't reviewing multiple changes together.